test/cpp/microbenchmarks/bm_callback_cq.cc

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
Is there a reason why you created this as a new file rather than just adding the new tests into the existing bm_cq.cc ? That would save a lot of build file changes/additions/etc